cpu/nrf52/ldscripts/nrf52832xxaa.ld

```diff
@@ -18,10 +18,14 @@
  * @}
  */
 
+rom_length = 512K;
```

Comment:
This variable will be redefined if the multislot strategy is used, you can look at [this](https://github.com/RIOT-OS/RIOT/pull/7209/files#diff-38f55d3264961035bd49433384e30db5R22) line where the variable is modified when an offset is defined. Otherwise it stays with the default value.